x-pack/plugins/files/server/feature.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DEFAULT_APP_CATEGORIES } from '@kbn/core-application-common';
+import { KibanaFeatureConfig } from '@kbn/features-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { PLUGIN_ID } from '../common';
+import { FILES_MANAGE_PRIVILEGE } from '../common/constants';
+import { hiddenTypes } from './saved_objects';
+
+// TODO: This should be registered once we have a management section for files content
+export const filesFeature: KibanaFeatureConfig = {
+  id: PLUGIN_ID,
+  name: i18n.translate('xpack.files.featureRegistry.filesFeatureName', {
+    defaultMessage: 'Files',
+  }),
+  minimumLicense: 'basic',
+  order: 10000,
+  category: DEFAULT_APP_CATEGORIES.management,
+  app: [PLUGIN_ID],
+  privilegesTooltip: i18n.translate('xpack.files.featureRegistry.filesPrivilegesTooltip', {
+    defaultMessage: 'Provide access to files across all apps',
```

Comment:
Shall it be added to all routes? Or, otherwise, shall this explicitly mention list/metric routes?

```suggestion
    defaultMessage: 'Provide access to read/find files across all apps',
```